levels/DogWorld/normal_02.gd

```diff
@@ -1,4 +1,5 @@
-#    Copyright (C) 2020  Rob Nugen
+#    Level 2
+#   Copyright (C) 2020 Rob Nugen
```

Comment:
Looks like the copyright line changed which will still work because it's a comment